tensorflow/lite/micro/kernels/xtensa/fully_connected.cc

```diff
@@ -28,6 +28,8 @@ limitations under the License.
 #include "tensorflow/lite/micro/kernels/xtensa/xtensa_fully_connected.h"
 #include "tensorflow/lite/micro/micro_log.h"
 
+#define FC_FIX_3D_DATA
+
```

Comment:
is this really necessary? can we simply change the default without introducing an ifdef?

Reply:
Yes, this #define is not necessary and the change can be default behavior.